diff --git a/lib/pages/common/slide/common_slide_page.dart b/lib/pages/common/slide/common_slide_page.dart
index 58ba68b7c..e17449ee1 100644
--- a/lib/pages/common/slide/common_slide_page.dart
+++ b/lib/pages/common/slide/common_slide_page.dart
@@ -142,6 +142,12 @@ class SlideDragGestureRecognizer extends HorizontalDragGestureRecognizer {
   bool isPointerAllowed(PointerEvent event) {
     return isDxAllowed(event.localPosition.dx) && super.isPointerAllowed(event);
   }
+
+  @override
+  void addAllowedPointer(PointerDownEvent event) {
+    super.addAllowedPointer(event);
+    resolve(.accepted);
+  }
 }
 
 class TabBarDragGestureRecognizer
